facturx/tests/test_facturx.py

```diff
@@ -1,6 +1,60 @@
 import os
 import unittest
-from facturx.facturx import FacturX
+from facturx.facturx import *
+import xml.etree.ElementTree as ET
+
+
+class XmlTree:
```

Comment:
- You can't [plagiarize](https://stackoverflow.com/questions/24492895/comparing-two-xml-files-in-python) a whole code block without license and credit.
- If a helper class is needed, it can't be in the test module.